src/components/settings/SignerAccountMFA/useMFASettings.ts

```diff
@@ -0,0 +1,32 @@
+import useMPC from '@/hooks/wallets/mpc/useMPC'
+import { COREKIT_STATUS } from '@web3auth/mpc-core-kit'
+
+export type MFASettings = {
+  mfaEnabled: boolean
+} | null
+
+const useMFASettings = () => {
```

Comment:
I couldn't find where this hook is being used. What is the purpose of it?

Reply:
Oh I think it's some legacy hook. I will merge the current epic branch in. That should remove this hook.